lang/en.json

```
@@ -1,9 +1,9 @@
 {
     "SecurityInfo": {
-        "PluginDescription": "Based on PhpSecInfo from the PHP Security Consortium, this plugin provides security information about your PHP environment and offers suggestions for improvement. It is a tool in a multilayered security approach. It does not replace secure development practices nor audit the code\/application.",
+        "PluginDescription": "Security info and suggested improvements for the PHP. As a multilayered security approach, it does not replace secure development practices, nor auditing the code\/application. Based on PhpSecInfo from the PHP Security Consortium.",
```

Personally I would prefer something like this:
```suggestion
        "PluginDescription": "Provides security info and suggests improvements for your PHP environment. As part of a multilayered security approach, it does not replace secure development practices, nor auditing the code\/application. Based on PhpSecInfo from the PHP Security Consortium.",
```